common/navigation-data.js

```diff
@@ -80,6 +80,13 @@ export const generate = ({ library = [], slates = [] }) => [
     pageTitle: "Slates",
     children: null,
   },
+  {
+    id: "V1_NAVIGATION_WILL_SCENE",
+    decorator: "SCENE_WILL",
+    name: "Will's Scene",
+    pageTitle: "Slate",
+    children: null,
```

Comment:
Will this work? Have you figured out how to make sure it doesn't appear on the sidebar?

Reply:
Yes, it ran locally, and to avoid it running in the sidebar I just add `ignore: true,`(line 89) in the next line.